src/html/components/mm/templates/content-accordion.handlebars

```diff
@@ -1,5 +1,5 @@
 {{> header}}
-<main id='contentTemplate'> 
+<main id='accordionTemplate'> 
```

Comment:
Should this be `buttonTemplate` ?

Reply:
I'm not sure. Do you mean it should be contentTemplate?

Reply:
Well I saw that you had `accordionTemplate` on the accordion example page you made, so following that idea, then this should be `buttonTemplate` or we could make all of the example pages also just use `contentTemplate`